tests/third_party/bullet/src/LinearMath/btSerializer.cpp

```diff
@@ -1,3 +1,6 @@
+// gcc 9.2.1 complains about narrowing
+#pragma GCC diagnostic ignored "-Wnarrowing"
```

Comment:
I though we already disable this warning (and many others when we build box2d).

At lest in the test_core version we add `self.emcc_args += ['-Wno-c++11-narrowing', '-Wno-deprecated-register', '-Wno-writable-strings']`

Reply:
True, but we also need this for the benchmarker when it runs the code natively. Not sure we have a nice way to share that across everything - perhaps editing the configure/make build system, but after looking for a while that seemed very inconvenient.